integration-tests/utils/integration-utils.js

```diff
@@ -334,7 +334,7 @@ const logPageError = page => {
 }
 
 const logPageConsoleMessages = page => {
-  const handle = msg => console.log('page.console', msg)
+  const handle = msg => console.log('page.console', msg.text())
```

Comment:
No more of this

```page.console ConsoleMessage {
  _type: 'error',
  _text: 'Failed to load resource: the server responded with a status of 404 ()',
  _args: [],
  _stackTraceLocations: [
    {
      url: 'https://firecloud-orchestration.dsde-dev.broadinstitute.org/api/nih/status',
      lineNumber: undefined
    }
  ]
}
```
It became this one line
```
page.console Failed to load resource: the server responded with a status of 404 ()
```

Reply:
The URL is very important to have logged though-- otherwise you don't know what server call resulted in the error.

Reply:
This is what looks like in this PR. I think it's easy to find out which request has failed since logs are printed in the correct order. Do you stil want me revert this?
```
age.http GET 404 https://broad-bond-dev.appspot.com/api/link/v1/anvil
page.console Failed to load resource: the server responded with a status of 404 ()
page.http GET 404 https://broad-bond-dev.appspot.com/api/link/v1/kids-first
page.console Failed to load resource: the server responded with a status of 404 ()
page.http GET 404 https://broad-bond-dev.appspot.com/api/link/v1/dcf-fence
page.console Failed to load resource: the server responded with a status of 404 ()
page.http GET 404 https://broad-bond-dev.appspot.com/api/link/v1/fence
page.console Failed to load resource: the server responded with a status of 404 ()
```

Reply:
I remembered reason. Some page.console msg are too long.
```
page.console ConsoleMessage {
  _type: 'warning',
  _text: 'actions jupyter-notebook:find-and-replace does not exist, still binding it in case it will be defined later...',
  _args: [
    JSHandle {
      _disposed: false,
      _context: [ExecutionContext],
      _client: [CDPSession],
      _remoteObject: [Object]
    },
    JSHandle {
      _disposed: false,
      _context: [ExecutionContext],
      _client: [CDPSession],
      _remoteObject: [Object]
    },
    JSHandle {
      _disposed: false,
      _context: [ExecutionContext],
      _client: [CDPSession],
      _remoteObject: [Object]
    }
  ],
  _stackTraceLocations: [
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/notebook/js/main.min.js?v=7e50cbb06bc00a057be483279bc938810b467e6c9dd9b5d91eb4b06b75704564224322da095d638da40ccb4cbe79a45d46de8eb77ac598b5c1c744d71ff0834f',
      lineNumber: 69955,
      columnNumber: 24
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/notebook/js/main.min.js?v=7e50cbb06bc00a057be483279bc938810b467e6c9dd9b5d91eb4b06b75704564224322da095d638da40ccb4cbe79a45d46de8eb77ac598b5c1c744d71ff0834f',
      lineNumber: 69735,
      columnNumber: 17
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/notebook/js/main.min.js?v=7e50cbb06bc00a057be483279bc938810b467e6c9dd9b5d91eb4b06b75704564224322da095d638da40ccb4cbe79a45d46de8eb77ac598b5c1c744d71ff0834f',
      lineNumber: 73468,
      columnNumber: 18
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1692,
      columnNumber: 32
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 880,
      columnNumber: 50
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1135,
      columnNumber: 33
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 133,
      columnNumber: 22
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1185,
      columnNumber: 20
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 58,
      columnNumber: 30
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1184,
      columnNumber: 16
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 935,
      columnNumber: 29
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1172,
      columnNumber: 21
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 785,
      columnNumber: 25
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1199,
      columnNumber: 62
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1586,
      columnNumber: 20
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1713,
      columnNumber: 28
    }
  ]
}
page.console ConsoleMessage {
  _type: 'log',
  _text: 'Loaded moment locale en',
  _args: [
    JSHandle {
      _disposed: false,
      _context: [ExecutionContext],
      _client: [CDPSession],
      _remoteObject: [Object]
    },
    JSHandle {
      _disposed: false,
      _context: [ExecutionContext],
      _client: [CDPSession],
      _remoteObject: [Object]
    }
  ],
  _stackTraceLocations: [
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/notebook/js/main.min.js?v=7e50cbb06bc00a057be483279bc938810b467e6c9dd9b5d91eb4b06b75704564224322da095d638da40ccb4cbe79a45d46de8eb77ac598b5c1c744d71ff0834f',
      lineNumber: 57864,
      columnNumber: 14
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1692,
      columnNumber: 32
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 880,
      columnNumber: 50
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1172,
      columnNumber: 21
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 785,
      columnNumber: 25
    },
    {
      url: 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/terra-dev-8f336543/saturn-7a813c4c-6d1b-4cff-885a-ec426686383a/static/components/requirejs/require.js?v=d37b48bb2137faa0ab98157e240c084dd5b1b5e74911723aa1d1f04c928c2a03dedf922d049e4815f7e5a369faa2e6b6a1000aae958b7953b5cc60411154f593',
      lineNumber: 1456,
      columnNumber: 35
    }
  ]
}
```